src/stratigraphy/util/find_description.py

```diff
@@ -194,6 +219,15 @@ def _block_separated_by_lefthandside_line_segment(
         )  # for the block splitting, we only care about x-extension
 
         line_ends_block = last_line_y_coordinate < line_y_coordinate and line_y_coordinate < current_line_y_coordinate
-        if is_line_long_enough and line_ends_block and line_cuts_lefthandside_of_block:
+
+        weak_condition = (
+            (line.start.x - 5 < block.rect.x0 and line.end.x > block.rect.x0)
+            and (np.abs(line.start.x - line.end.x) > length_threshold - 2)
+            and (len(block.lines) > 1)
+        )  # if block has at least three lines, we weaken the splitting condition.
```

Comment:
Could you elaborate on the intuition behind this weaker condition? Why does it make sense to weaken the splitting condition if the block has at least three lines?

Reply:
Looking at the pdf's, and the errors we made, I have realised that it seldomly happens to have more than three lines in one block. If blocks are larger, then, more often than not, we missed a split. That lead me to the decision to experiment with weaker conditions for large blocks only.

Afterwards, I checked all the documents that changed due to this weaker condition, and the improvements are dominant. Especially in some cases there is a large improvement (i.e. many more cases are recognized) whereas a few documents do not benefit from this change.